dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -162,10 +175,30 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 			}
 		}
 		params = sortByKey(params);
-
+		boolean byInode = params.containsKey("byInode");
+		// Default to a no-shortyId value
+		ShortyId shorty = shortyIdApi.noShorty(uuid);
+		try {
+			// Try to get the ShortyId from a valid expected UUID value
+			final Optional<ShortyId> shortOpt = shortyIdApi.getShorty(uuid);
+			if (shortOpt.isPresent()) {

```

Comment:
I think you can avoid this call if you just do 
```
if(!byInode) { 
```

B/c if byInode is true, does not matter which value is on shorty

Reply:
Not really. Both Inodes and Identifiers can have the legacy format, i.e., short numeric-only values. We need to catch both cases and make sure we bypass the UUID format validation.